app/javascript/gobierto_dashboards/webapp/containers/shared/Aside.vue

```diff
@@ -30,6 +30,17 @@
           </div>
         </Dropdown>
       </div>
+
+      <div
+        v-if="showDownloadDataLink()"
```

Comment:
Giving another look at this, it looks weird to me.
So, you're calling a function in the DOM (bad practice) to get a Boolean. This boolean is about `dataDownloadEndpoint` whether exists or not.
`dataDownloadEndpoint` is a prop, and it has a default value, so always gonna exist, at least empty. 
When does `dataDownloadEndpoint` change? I presume never in the life. In such case, why is not declarated in `created` lifecycle? I'm saying:

```
created() {
   this.isShowDownloadDatalinkShown = !!dataDownloadEndpoint
}
```

End of history.

What if the `dataDownloadEndpoint` prop changes?

```
watch: {
  dataDownloadEndpoint(old, new) {
    if (old !== new) {
      this.isShowDownloadDatalinkShown = !!new
    }
  }
}
```

But I don't think that be the case, but keep in mind for future approachs

Reply:
Yeah, makes mucho more sense. I changed it (added the watcher too).